packages/apps-config/src/api/spec/polkadex.ts

```diff
@@ -12,72 +12,35 @@ const definitions: OverrideBundleDefinition = {
       // on all versions
       minmax: [0, undefined],
       types: {
-        OrderType: {
+        chainbridge::ChainId: 'u8',
```

Comment:
CI is not happy here - https://github.com/polkadot-js/apps/pull/5492/checks?check_run_id=2770449568#step:3:557

Reply:
Resolved issues :)